projects/extension/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Changed
 
 - Update @substrate/smoldot-light to [version 0.6.8](https://github.com/paritytech/smoldot/blob/main/bin/wasm-node/CHANGELOG.md#068---2022-03-23) ([#890](https://github.com/paritytech/substrate-connect/pull/890))
+- The smoldot background worker will now bound its CPU usage to 50% of one CPU on average. ([#900](https://github.com/paritytech/substrate-connect/pull/900))
```

Comment:
Im not sure if this is something that end user should know. Its extension-internal - right?

Reply:
It's visible to the user that the usage of their CPU is bounded to 50%